src/Playwright/Core/AssertionsBase.cs

```diff
@@ -140,5 +146,14 @@ private string FormatValue(object value)
 
             return value.ToString();
         }
+
+        internal static void SetDefaultTimeout(float? timeout)
+        {
+            if (timeout == null)
+            {
+                return;
+            }
+            defaultTimeout = timeout.Value;
+        }
```

Comment:
Maybe null should set the default? 
Otherwise, we could inline this.

```suggestion
        internal static void SetDefaultTimeout(float? timeout)
            => _defaultTimeout =  timeout == null ? _defaultTimeoutValue : timeout;
```